charts/rancher-cis-benchmark/0.1.3/values.yaml

```diff
@@ -30,7 +30,7 @@ image:
     tag: v0.1.14
```

Comment:
The chart.yaml version here is still 0.1.2. Since this is going into 2.6 you should bump the version to 0.2.0.

Reply:
Ok, changed to 0.2.0